src/tlo/methods/bed_days.py

```diff
@@ -145,6 +145,40 @@ def initialise_beddays_tracker(self, model_to_data_popsize_ratio=1.0):
             assert not df.isna().any().any()
             self.bed_tracker[bed_type] = df
 
+    def switch_beddays_availability(
+        self,
+        new_availability: Literal["all", "none", "default"],
+        effective_on_and_from: Date,
+        model_to_data_popsize_ratio: float = 1.0,
```

Comment:
it would be nicer (I think) if the user didn't need to pass this. It hasn't changed since the beginning of the simulation, so it seems awkward to pass it in just to instruct a change in availability to all/none/default.